src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelBindingContext.cs

```diff
@@ -232,69 +82,93 @@ public ModelStateDictionary ModelState
         /// <remarks>
         /// The <see cref="ModelMetadata"/> property must be set to access this property.
         /// </remarks>
-        public Type ModelType => ModelMetadata?.ModelType;
+        public abstract Type ModelType { get; }
 
         /// <summary>
-        /// Gets or sets a model name which is explicitly set using an <see cref="IModelNameProvider"/>.
-        /// <see cref="Model"/>.
+        /// Represents the <see cref="ModelBinding.OperationBindingContext"/> associated with this context.
```

Comment:
> prefer to defer the copy edits for an API update sweep.
> FYI if you fixed this, would be no need for a follow-up pass on this file.

To speak to this, there is no planned API update sweep I've heard of.

At the same time, this is an extensibility point used by users at a medium frequency. If there are issues caused by the changes in this PR we should fix them.

This isn't the place to do a broad pass on all the existing docs for formatting. Open a separate issue if that's needed so it doesn't get lost.